src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -577,7 +577,12 @@ bool clang_c_convertert::get_function(const clang::FunctionDecl &fd, exprt &)
   // Don't convert if implicit, unless it's a constructor
```

Comment:
update this comment for destructor as well

Reply:
Done